persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/spring/data/jpa/query/UserRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,162 @@
+package com.baeldung.spring.data.jpa.query;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
```

Comment:
Is there a reason not to use JUnit 5?

Reply:
Well none of the tests in this module use JUnit 5 (yet). So I think it's better to stick to the pattern for now and create a new task to upgrade all the tests to JUnit 5.

Reply:
sounds good @kwoyke 👍